documentation/tutorial.md

```diff
@@ -36,8 +36,6 @@ This will create a tutorial directory with the following contents:
 ```bash
 tutorial
 ├── README.md
-├── config
```

Comment:
Maybe this should not be removed?

Reply:
`mix new` doesn't create this directory anymore, that's why I removed it here and added instructions to create the config file manually.